### PR TITLE
Initial public release

### PR DESCRIPTION
**Pull request by FinalAngel**
Allows to create ad-hoc plugins from a plugin set.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    cms36: https://github.com/divio/django-cms/archive/release/3.6.x.zip
+commands =
+    {envpython} --version
+    {env:COMMAND:coverage} erase
+    {env:COMMAND:coverage} run setup.py test
+    {env:COMMAND:coverage} report
+
+[flake8]
+max-line-length = 120
+exclude = */docs/*,*/migrations/*
```
Comment (vThaian) [inline tox.ini]: please exclude also .tox/

**Comment by vThaian** [comment]
I think that adding a test with checking isort would be great to have in addons. What do you think @FinalAngel ?

**Comment by vThaian** [comment]
The second suggestion is to start adding `.DS_Store` to `.gitignore` and `MANIFEST` file

File: README.rst [hunk 2]
```diff
@@ -3,9 +3,61 @@ django CMS Modules
...
+
+Documentation
+=============
+
+See ``REQUIREMENTS`` in the `setup.py <https://github.com/divio/djangocms-modules/blob/master/setup.py>`_
+file for additional dependencies:
+
+* Python 2.7, 3.4 or higher
+* Django 1.11 or higher
+* django CMS 3.5 or higher
```
[the remark this answers is not in the document]
Comment (FinalAngel) [reply README.rst]: This addon is only compatible with >=3.5 @vThaian

File: setup.py [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
 from setuptools import setup, find_packages
 
-import djangocms_modules
+from djangocms_modules import __version__
+
+
+REQUIREMENTS = [
+    'django-cms>=3.5.0',
```
[the remark this answers is not in the document]
Comment (FinalAngel) [reply setup.py]: Not for this one

**Comment by FinalAngel** [comment]
@vThaian "checking for isort" > Do you have an example?
@vThaian ".DS_Store" > I agree adding it to .gitignore but not the MANIFEST

File: tox.ini (hunk 1 above)
Comment (FinalAngel) [reply tox.ini]: I copied this file from addon to addon, if thats the case, I can mark to update all of them at once. Getting back at a "configuration" addon

**Pull request merged**